src/Advection/weno5.jl

```diff
@@ -0,0 +1,274 @@
+#####
+##### Weighted Essentially Non-Oscillatory (WENO) scheme of order 5
+#####
+
+struct WENO5 <: AbstractAdvectionScheme end
+
+#####
+##### ENO interpolants of size 3
+#####
+
+@inline px₀(i, j, k, f) = @inbounds + 1/3 * f[i-1, j, k] + 5/6 * f[i,   j, k] -  1/6 * f[i+1, j, k]
```

Comment:
the variable name `f` is used throughout. At first, I thought `f` might be a function. In fact, it is a variable, correct? In other contexts we have used either `u, v, w, c`, and sometimes `ψ` (to denote a variable that's either velocity or tracer, since `φ` is taken by the pressure-like quantity). Using one of these names might make this code more understandable on first impression. (Doc strings could help too)

Reply:
Changed to `ψ`.